From f9a26bebde2f4c225545666242c81066ae5cb084 Mon Sep 17 00:00:00 2001
From: Udit Kumar <u-kumar1@ti.com>
Date: Sat, 7 Dec 2024 12:42:54 +0530
Subject: [PATCH 1/2] Revert "HACK: arm: dts: am57: disable late attach as
 default"

Build support added in Yocto,
So revert commit c490f4c0fe51281818c45159c0fbed94f852978e.

Signed-off-by: Udit Kumar <u-kumar1@ti.com>
---
 arch/arm/boot/dts/am574x-idk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/am574x-idk.dts b/arch/arm/boot/dts/am574x-idk.dts
index 88a4c9b31a5a3..9eeed5f8056e8 100644
--- a/arch/arm/boot/dts/am574x-idk.dts
+++ b/arch/arm/boot/dts/am574x-idk.dts
@@ -48,3 +48,4 @@ &pruss2_mdio {
 	status = "disabled";
 };
 
+#include "dra7-ipu-common-early-boot.dtsi"
-- 
2.47.0

